aim/api/resource.py

```diff
@@ -86,7 +89,6 @@ class Tenant(AciResourceBase):
 
     identity_attributes = ['name']
     other_attributes = ['display_name']
-    db_attributes = []
 
     _aci_mo_name = 'fvTenant'
     _tree_parent = None
```

Comment:
Should the Neutron (and eventually GBP) ensure_tenant() stuff be creating Tenant resources if they don't already exist?